### ISSUE TITLE
Process output empty

### ISSUE THREAD

**Issue by basz** [symfony/symfony#38256]
**Symfony version(s) affected**: x.y.z

Symfony/process                      v5.1.5

Using a docker image phpswoole/swoole:4.4.16-php7.4

**Description**  
Running a process inside a Swoole worker yields no output.

**How to reproduce**  
I have created a mezzio-skeleton application and enhanced it with mezzio-swoole. Inside a worker (simple handler) I run the following;

```
$process = new Process(['ls', '-la']);
$process->run();
$process->getOutput(); // --> \n
```

whereas 

```
exec('ls -la', $output);
var_dump($output); // yields expected output
```

**Possible Solution**  
It has probably has to to with Swoole interfering with `stream_get_contents($this->stdout, -1, 0)`...
I suspect this because running a Process directly with the PHP executable does yield output from the executable.

**Additional context**

**Comment by basz** [comment]
Interestingly using `FROM phpswoole/swoole:4.4.16-php7.3` as my docker image resolves the issue.

**Comment by stof** [comment]
Can you give the exact PHP version for the working and non working cases ?

**Comment by basz** [comment]
```
PHP 7.3.15 (cli) (built: Feb 26 2020 12:33:18) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.3.15, Copyright (c) 1998-2018 Zend Technologies
```

**Comment by basz** [comment]
```
PHP 7.4.3 (cli) (built: Feb 26 2020 12:05:30) ( NTS )
Copyright (c) The PHP Group
Zend Engine v3.4.0, Copyright (c) Zend Technologies
```

**Comment by stof** [comment]
7.4.3 is quite old. The latest version is 7.4.10.

And I know that PHP 7.4 contained some regressions so it might be fixed in newer PHP versions.

**Comment by basz** [comment]
fixed by php updating to 7.4.10
